### PR TITLE
Merge for cmd error

### PR DESCRIPTION
**Pull request by JohnFawkes**

### CODE REVIEW

**Comment by lzlrd** [comment]
Looks good to me!

**Comment by lzlrd** [comment]
Just add a version increment and update changelog before you merge.

File: common/service.sh [hunk 1]
```diff
@@ -5,3 +5,5 @@ MODDIR=${0%/*}
 
 # This script will be executed in late_start service mode
 # More info in the main Magisk thread
+am force-stop "com.google.android.dialer"
```
Comment (thehappydinoa) [inline common/service.sh:8]: Do we need to force stop on reboot?

Comment (JohnFawkes) [reply common/service.sh:8]: According to @Zackptg5 yes we do

Comment (Zackptg5) [reply common/service.sh:8]: I just suggested that the am command gets moved to late start service script since post-fs-data runs before am will even work

Comment (JohnFawkes) [reply common/service.sh:8]: That's what I meant.

**Review by ThatNerdyPikachu — APPROVED** [review]
LGTM!

**Review by thehappydinoa — APPROVED** [review]
Looks good to me

**Pull request merged**